src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -296,6 +296,8 @@ void notifyChatRoomMemberLeft(ChatRoomMember chatMember)
     void notifyChatRoomMemberUpdated(ChatRoomMember chatMember, Presence presence)
     {
         super.notifyChatRoomMemberUpdated(chatMember, presence);
+        String identifier = getParticipantIdentifier(chatMember);
+        this.transcriber.updateParticipant(identifier, chatMember);
```

Comment:
Can you add a comment why is this done. ... I guess to update a language change coming in the presence ...